src/app/components/mno-apps/mno-app.coffee

```diff
@@ -38,13 +38,24 @@ angular.module 'mnoEnterpriseAngular'
 
     vm.pricedPlan = ProvisioningHelper.pricedPlan
 
-    getPricingPlans = () ->
+    # Ordering is possible if
+    # -Plans exist in the current currency
+    # -The app only has free or pay as you go plans
+    # -Price selection can be skipped
+    orderPossible = (plans, product) ->
+      if product
+        vm.orderPossible = plans.default[0] || ProvisioningHelper.skipPriceSelection(product)
+      else
+        vm.orderPossible = plans.default[0]
+
+    getPricingPlans = (product) ->
```

Comment:
#### For this `getPricingPlans` function:
```coffeescript
getPricingPlans = (product) ->
  if vm.isPriceShown
    plans = vm.app.pricing_plans
    plans_for_currency = ProvisioningHelper.planForCurrency(plans, vm.currency)
    if _.isEmpty(plans_for_currency)
      orderPossible(plans, product)
    else
      vm.pricing_plans = plans_for_currency
      vm.orderPossible = true
```
This will ensure that any 'payg' or 'free' plans are displayed regardless of what currency is set.
I think this is going to resolve a few pricing issues 🥇